src/pages/about.vue

```diff
@@ -6,75 +6,81 @@
             </template>
           </i18n>
+
           <i18n path="about.planning.content" tag="p">
-            <template #vision>
-              <a
-                :aria-label="$t('about.aria.vision')"
-                href="https://creativecommons.org/2019/03/19/cc-search/"
-                >{{ $t('about.planning.vision') }}</a
-              >
-            </template>
-            <template #roadmap>
-              <a
-                :aria-label="$t('about.aria.roadmap')"
-                href="https://docs.google.com/document/d/19yH2V5K4nzWgEXaZhkzD1egzrRayyDdxlzxZOTCm_pc/edit#heading=h.jih78emira0r"
-                >{{ $t('about.planning.roadmap') }}</a
-              >
-            </template>
-            <template #working>
-              <a
-                :aria-label="$t('about.aria.projects')"
-                href="https://github.com/orgs/WordPress/projects/3"
-                >{{ $t('about.planning.working') }}</a
+            <template #meta>
+              <NuxtLink
+                :aria-label="$t('about.aria.meta')"
+                :to="localePath('/meta-search')"
+                >{{ $t('about.planning.meta') }}</NuxtLink
               >
             </template>
             <template #search>
               <a
-                aria-label="Openverse repository"
+                :aria-label="$t('about.aria.search')"
                 href="https://github.com/wordpress/openverse-frontend/"
                 >{{ $t('about.planning.search') }}</a
               >
             </template>
-            <template #catalog-api>
+            <template #api>
               <a
-                aria-label="cc catalog-api repository"
+                :aria-label="$t('about.aria.api')"
                 href="https://github.com/wordpress/openverse-api/"
-                >{{ $t('about.planning.catalog-api') }}</a
+                >{{ $t('about.planning.api') }}</a
               >
             </template>
             <template #catalog>
               <a
-                aria-label="cc catalog repository"
+                :aria-label="$t('about.aria.catalog')"
                 href="https://github.com/wordpress/openverse-catalog/"
                 >{{ $t('about.planning.catalog') }}</a
               >
             </template>
-            <!-- <template #community>
+            <template #community>
+              <!-- TODO: Update link to team page on Make WordPress -->
               <a
-                :aria-label="$t('about.aria.contribution')"
-                href="https://creativecommons.github.io/contributing-code/"
+                :aria-label="$t('about.aria.community')"
+                href="https://make.wordpress.org/"
                 >{{ $t('about.planning.community') }}</a
               >
-            </template> -->
+            </template>
+            <template #working>
+              <a
+                :aria-label="$t('about.aria.projects')"
+                href="https://github.com/orgs/WordPress/projects/3"
```

Comment:
@zackkrida the GitHub project is private. If we link to it, do we need to make it public?

Reply:
It's been made public, thanks for pointing this out.